src/usdb_syncer/gui/settings_dialog.py

```diff
@@ -181,4 +258,24 @@ def _save_settings(self) -> bool:
                 self, "Invalid setting", "Please provide a valid path template!"
             )
             return False
+        settings.set_app_path(
+            settings.SupportedApps.KAREDI, self.lineEdit_path_karedi.text()
+        )
+        settings.set_app_path(
+            settings.SupportedApps.PERFORMOUS, self.lineEdit_path_performous.text()
+        )
+        settings.set_app_path(
+            settings.SupportedApps.ULTRASTAR_MANAGER,
+            self.lineEdit_path_ultrastar_manager.text(),
+        )
+        settings.set_app_path(
+            settings.SupportedApps.USDX, self.lineEdit_path_usdx.text()
+        )
+        settings.set_app_path(
+            settings.SupportedApps.VOCALUXE, self.lineEdit_path_vocaluxe.text()
+        )
+        settings.set_app_path(
+            settings.SupportedApps.YASS_RELOADED,
+            self.lineEdit_path_yass_reloaded.text(),
+        )
```

Comment:
Might be nice to do some validation, i.e. check if the pass exists. Alternatively, make the text edits readonly so they can only be set to a valid path.